docs/content/en/docs/concepts/components/core/executor.md

```diff
@@ -11,8 +11,8 @@ Executor is the component to spin up function pods for functions. When Router re
 requests to a function, it checks whether a function service record exists in its cache. 
 If cache misses, the function service record was found or expired, it asks Executor to 
 provide a new one. Executor then retrieves function information from Kubernetes CRD and 
-invokes one of the executor types to spin up function pods. Once the function pods up, 
-a function service record contains the address of a service/pod will be returned.
+invokes one of the executor types to spin up function pods. Once the function pods are up, 
+a function service record contains the address of a service/pod that will be returned.
```

Comment:
I think `a function service record that contains the address of a service/pod will be returned.` would be better?

Reply:
Sure. I'll change it.